utils/docker/Dockerfile.ubuntu-plugins-deps

```diff
@@ -60,13 +60,17 @@ RUN [ "/bin/bash", "install-pytorch.sh"  ]
 ARG UBUNTU_VER
 
 COPY wasi-nn/install-openvino.sh .
+COPY wasi-nn/install-openvino-genai.sh .
 ENV OPENVINO_UBUNTU_VERSION=${UBUNTU_VER}
 ENV OPENVINO_VERSION="2025.0.0"
 ENV OPENVINO_YEAR="2025"
+ENV OPENVINOGEN_VERSION="2025.0.0.0"
+ENV OPENVINOGEN_YEAR="2025.0"
+RUN [ "/bin/bash", "install-openvino.sh" ]
 RUN [ "/bin/bash", "install-openvino.sh" ]
 
 COPY wasi-nn/install-onnxruntime.sh .
-RUN [ "/bin/bash", "install-onnxruntime.sh" ]
```

Comment:
Why did you delete the ONNX Runtime script here? Was it a typo or something?

Reply:
fixed